VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/custom/CustomSelectViewAction.java

```diff
@@ -0,0 +1,38 @@
+package org.edx.mobile.test.feature.custom;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+
+import org.edx.mobile.view.custom.CustomSelectView;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static org.hamcrest.Matchers.allOf;
+
+/**
+ * Defines the custom {@link ViewAction} implementation to select an item
+ * from our custom implementation of spinner i.e. {@link CustomSelectView}.
+ */
+public class CustomSelectViewAction {
+    public static ViewAction selectSpinnerItem(final int index) {
+        return new ViewAction() {
```

Comment:
Also, instead of returning an anonymous subclass of `ViewAction` here, you might consider implementing it in the main `CustomSelectViewAction` itself. The name certainly recommends it, and otherwise it seems pretty useless. The constructor could remain private, and only accessible via this static factory method, to enable the wrapping with the assertions action wrapper.